tests/unit_tests/test_wave_03.py

```diff
@@ -131,7 +131,5 @@ def test_swap_items_from_their_empty_returns_false():
 
     result = fatimah.swap_items(jolie, item_b, nobodys_item)
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert len(jolie.inventory) == 0
+    assert len(fatimah.inventory) == 3
```

Comment:
Again, this is another place where you could check if the original items are inside of inventory still.